### DISCUSSION TITLE
Welcome to JSS-01 Discussions!

### DISCUSSION THREAD

**Discussion by michaelkolesidis** [michaelkolesidis/javascript-software-synthesizer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamjonny** [comment]
Hi! Just found this as still researching examples I can try to learn from or recycle or even contribute to. Is VS Code an adequate tool?
Great stuff!!! inspired 😎

Using jd-xi hoping to make some controllers to reduce menu diving and make it easier to understand what's occurring when synthesizing!

With gpt I was able to make a primitive fader that appeared to send the right sysex message based on the dynamic value from the fader which was modulated by an lfo with rate and depth, but then gave up after all subsequent efforts to align the controllers failed!

Hoping to get to something like [this](https://midisurf.app/) (ambitious ; )
